### PR TITLE
PIP-61: airflow variables(daily search task)

### PR DESCRIPTION
**Pull request by hbmshemet**

### CODE REVIEW

File: sentinel_search_dag.py [hunk 1]
```diff
@@ -15,14 +27,14 @@
...
-                                        resources={"limit_cpu": "300m", "limit_memory": "500Mi"},
-                                        startup_timeout_seconds=3600,
-                                        execution_timeout=timedelta(minutes=60),
-                                        retries=1,
-                                        retry_delay=timedelta(minutes=5),
+                                        resources={"limit_cpu": limit_cpu, "limit_memory": limit_memory},
+                                        startup_timeout_seconds=startup_timeout_seconds,
+                                        execution_timeout=timedelta(minutes=execution_timeout),
+                                        retries=retries,
+                                        retry_delay=timedelta(minutes=retries),
```
Comment (mshalak-nix) [inline sentinel_search_dag.py]: this should use retry_delay

File: sentinel_daily_search_dag.py [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
 from airflow import DAG
 from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
+from airflow.models import Variable
+
+config = Variable.get("sentinel_daily_search_config", deserialize_json=True)
+
+schedule_interval = config.get("schedule_interval", default_var="0 12 * * *")
+days_interval = config.get("days_interval", default_var=3)
+airflow_url = config.get("airflow_url", default_var="http://airflow.sandbox.hummingbird-tech.io/")
+product_bucket = config.get("product_bucket", default_var="mshalak")
```
Comment (mshalak-nix) [inline sentinel_daily_search_dag.py]: What happens if we do not specify defaults? Can we omit them for bucket and URL?

File: sentinel_daily_search_dag.py [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
+          schedule_interval=schedule_interval,
           start_date=datetime(2020, 1, 1),
           catchup=False)
 
-search_request = '{"days_interval" : 3, "airflow_url": "http://airflow.staging.hummingbird-tech.io/", ' \
-              '"cluster_airflow_url" : "http://172.21.25.7:8080/",' \
-              '"product_bucket" : "sentinel-cache-staging", "country_codes" : ["GBR"] }'
+search_request = f'{{"days_interval" : {days_interval}, "airflow_url": {airflow_url}", ' \
+              '"cluster_airflow_url" : "http://172.19.25.188:8080/",' \
+              f'"product_bucket" : {product_bucket}, "country_codes" : ["GBR"] }}'
```
Comment (mshalak-nix) [inline sentinel_daily_search_dag.py]: I guess country_codes are not used right now

File: sentinel_daily_search_dag.py (hunk 2 above)
Comment (hbmshemet) [reply sentinel_daily_search_dag.py]: yes, we can
it will return None

File: sentinel_daily_search_dag.py (hunk 3 above)
Comment (hbmshemet) [reply sentinel_daily_search_dag.py]: removed

File: sentinel_search_dag.py (hunk 1 above)
Comment (hbmshemet) [reply sentinel_search_dag.py]: done

File: sentinel_daily_search_dag.py [hunk 4]
```diff
@@ -1,16 +1,31 @@
...
-          schedule_interval="0 12 * * *",
+          schedule_interval=schedule_interval,
           start_date=datetime(2020, 1, 1),
           catchup=False)
 
-search_request = '{"days_interval" : 3, "airflow_url": "http://airflow.staging.hummingbird-tech.io/", ' \
-              '"cluster_airflow_url" : "http://172.21.25.7:8080/",' \
-              '"product_bucket" : "sentinel-cache-staging", "country_codes" : ["GBR"] }'
+search_request = f'{{"days_interval" : {days_interval}, "airflow_url": {airflow_url}", ' \
+              '"cluster_airflow_url" : "http://172.19.25.188:8080/",' \
```
Comment (mshalak-nix) [inline sentinel_daily_search_dag.py]: Don't forget to remove this IP or change it to staging's one

**Pull request merged**